rca/project_styleguide/templates/patterns/molecules/streamfield/stream_block.html

```diff
@@ -1,5 +1,5 @@
 {% load wagtailcore_tags %}
 
 {% for block in value %}
-    {% include_block block with guide_page=guide_page %}
+    {% include_block block %}
```

Comment:
presuming this wasn't needed for the guide pages then?

Reply:
I pulled down the live content and tested 5+ guide pages and they all matched live, I'm still a little confused as to what it did ;D